product/product.css

```diff
@@ -0,0 +1,70 @@
+.navbar{
+    padding: 0.25rem;
+ }
+ .search-container{
+    flex-basis: 25%;
+    border-radius: 1.5rem;
+    background: var(--white)
+ } 
+.search-bar{
+    outline: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    padding: 0.6rem 0.8rem;
+    margin: 0.5rem;
+}
+/* single card styling  */
+.single-card-container {
+    margin-top: 5rem;
+    height: 100vh;
+}
+.single-card{
+    display: flex;
+    box-shadow: var(--dark, 0.35 )0 0.25rem 1rem;
+    padding: 1.5rem;
+}
+.single-card-img{
+    width: 15rem;
+    height: 20rem;
+}
+.single-card-right{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
You can make a class of it and use it whenever needed.

Reply:
got it, made the changes! thanks for pointing out !!